app/models/feedback_submission.rb

```diff
@@ -1,18 +1,32 @@
-class FeedbackSubmission < ActiveRecord::Base
-  validates :body, presence: true
-  validate :validate_email
+class FeedbackSubmission
+  include NonPersistedModel
 
-  before_validation :strip_email_address, on: :create
+  attribute :body, String
+  attribute :email_address, String
```

Comment:
@mloughran sounds good to me about the feedback email.

This will be a problem for some visitors. If they use an invalid email for pvb2 then they'll get an error page.

We keep bumping into Sendgrid validations, would be good to make a final decision on what to do with them.  I'll add a card on the board for this.